doc/stdlib/base.rst

```diff
@@ -1545,7 +1565,7 @@ Internals
        julia> M.f()
        (1,2)
 
-   With @macroexpand the expression expands where @macroexpand appears in the code (module M). With macroexpand the expressions expands in the current module where the code was finally called. Note that when calling macroexpand or @macroexpand directly from the REPL, both of these contexts coincide, hence there is no difference.
+   With ``@macroexpand`` the expression expands where ``@macroexpand`` appears in the code (module ``M``\ ). With ``macroexpand`` the expressions expands in the current module where the code was finally called (REPL). Note that when calling ``macroexpand`` or ``@macroexpand`` directly from the REPL, both of these contexts coincide, hence there is no difference.
```

Comment:
ref https://github.com/JuliaLang/julia/pull/18660#discussion_r81490854 I think we should delete this `(REPL)` parenthetical both here and at the source, since it's confusing and not very general.